mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -1131,7 +1133,8 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT <p>This is saved on the {@code Invulnerable} NBT key.
 		COMMENT
 		COMMENT @implNote Invulnerable entities are immune from all damages except {@link
-		COMMENT DamageSource#OUT_OF_WORLD} and damages by creative mode players by default.
+		COMMENT net.minecraft.entity.damage.DamageSources#outOfWorld}
```

Comment:
Should probably reference the type `DamageTypes.OUT_OF_WORLD` rather than the source, as potentially one could create another source using that type.

Reply:
```suggestion
		COMMENT net.minecraft.entity.damage.DamageTypes#OUT_OF_WORLD}
```